dags/socorro_import.py

```diff
@@ -2,8 +2,12 @@
 from airflow.operators.subdag_operator import SubDagOperator
 from airflow.contrib.hooks.aws_hook import AwsHook
 from airflow.contrib.hooks.gcp_api_base_hook import GoogleCloudBaseHook
-from airflow.contrib.operators.bigquery_table_delete_operator import BigQueryTableDeleteOperator # noqa
-from airflow.contrib.operators.gcp_transfer_operator import S3ToGoogleCloudStorageTransferOperator  # noqa
+from airflow.contrib.operators.bigquery_table_delete_operator import (
+    BigQueryTableDeleteOperator,
+)  # noqa
```

Comment:
```suggestion
)
```
I guess this exception is no longer needed after reformat?